widgets/src/components/TaskList/TaskListContainer.js

```diff
@@ -191,8 +231,8 @@ class TaskList extends React.Component {
     return (
       <ThemeProvider theme={theme}>
         <Paper className={classes.paper}>
-          {error ? (
-            <ErrorComponent message={i18next.t('messages.errors.dataLoading')} />
+          {blocker ? (
```

Comment:
can't we simplify that with a single state? Managing only the `blockerMessage` will be enough here.

Reply:
i have remove `blockerMessage` and modified `blocker` from bool into the new `blockerMessage`